### ISSUE TITLE
Add sample data files for development

### ISSUE THREAD

**Issue by glauberramos** [datasciencebr/jarbas#120]
It would be nice to have some sample files to import into the database in order to do development of the app (elm, css, etc). It does not need to be the latest data, just some data to work on.

**Comment by cuducos** [comment]
Great idea!

**Comment by gomex** [comment]
Hey @cuducos 

I tried to use sample data from #121 PR, but I got error:

```
# ls contrib/sample-data/companies_sample.xz
contrib/sample-data/companies_sample.xz
# python manage.py reimbursements contrib/sample-data/companies_sample.xz
Starting with 0 reimbursements
Traceback (most recent call last):
  File "manage.py", line 22, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.5/site-packages/django/core/management/__init__.py", line 367, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.5/site-packages/django/core/management/__init__.py", line 359, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.5/site-packages/django/core/management/base.py", line 294, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.5/site-packages/django/core/management/base.py", line 345, in execute
    output = self.handle(*args, **options)
  File "/code/jarbas/core/management/commands/reimbursements.py", line 27, in handle
    self.bulk_create_by(self.reimbursements, options['batch_size'])
  File "/code/jarbas/core/management/commands/reimbursements.py", line 85, in bulk_create_by
    for reimbursement in reimbursements:
  File "/code/jarbas/core/management/commands/reimbursements.py", line 35, in reimbursements
    yield Reimbursement(**self.serialize(row))
  File "/code/jarbas/core/management/commands/reimbursements.py", line 49, in serialize
    reimbursement[new] = reimbursement[old]
KeyError: 'subquota_number'
```

**Comment by cuducos** [comment]
Hey @gomex, can you share the output of `python manage.py showmigrations`?

**Comment by gomex** [comment]
```
root@32651dbbee74:/code# python manage.py showmigrations
admin
 [X] 0001_initial
 [X] 0002_logentry_remove_auto_add
auth
 [X] 0001_initial
 [X] 0002_alter_permission_name_max_length
 [X] 0003_alter_user_email_max_length
 [X] 0004_alter_user_username_opts
 [X] 0005_alter_user_last_login_null
 [X] 0006_require_contenttypes_0002
 [X] 0007_alter_validators_add_error_messages
 [X] 0008_alter_user_username_max_length
contenttypes
 [X] 0001_initial
 [X] 0002_remove_content_type_name
core
 [X] 0001_initial
 [X] 0002_add_indexes
 [X] 0003_remove_some_indexes
 [X] 0004_add_receipt_url
 [X] 0005_add_receipt_url_last_update
 [X] 0006_lazy_backend_receipt_url
 [X] 0007_add_suppliers
 [X] 0008_optimize_char_field_lengths
 [X] 0009_add_latitude_and_longitude
 [X] 0010_extract_receipt
 [X] 0011_subquota_description_length
 [X] 0012_use_date_not_datetime
 [X] 0013_create_model_reimbursement
 [X] 0014_add_suspicions_and_probability_to_reimbursements
 [X] 0015_add_receipt_to_reimbursement
 [X] 0016_add_custom_ordering_to_reimbursement
 [X] 0017_make_some_fields_optional
 [X] 0018_make_issue_date_required
 [X] 0019_cleanup_remove_old_api
 [X] 0020_rename_supplier_to_company
 [X] 0021_make_reciept_fetched_a_db_index
sessions
 [X] 0001_initial
```

**Comment by cuducos** [comment]
You're using the wrong command/file combination — i.e.:

* The `python manage.py `**` reimbursements`** command
* with the file the `contrib/sample-data/`**`companies`**`_sample.xz`

; )

**Comment by gomex** [comment]
Done! #127 created

**Comment by cuducos** [comment]
Closed by #121
